### DISCUSSION TITLE
现在用mpv播放在线链接，怎么配置为yt-dlp使用aria2c呢？

### DISCUSSION THREAD

**Discussion by Arc-2023** [dyphire/mpv-config#39] [Q&A] [answered]
看那个hook文件已经删除了
以下是我的配置:
```conf
ytdl=yes
ytdl-raw-options-append=proxy=http://127.0.0.1:10809
ytdl-raw-options-append=external-downloader=aria2c
ytdl-raw-options-append=downloader-args=aria2c:"-x 256 -k 128k"
```
因为有个aria2c的版本支持无限大的线程，下载缓冲都非常快
这样配置之后缓冲速度还是没裸的yt-dlp使用aria2c下载快
怀疑是还没用上aria2c

**Comment by zhongfly** [comment, marked as answer]
https://github.com/dyphire/mpv-config/discussions/6 的重复
简而言之，对mpv来说，ytdl只是负责解析在线网站地址以获得真实的视频文件链接，mpv不使用ytdl下载文件，你怎么换aria2c/改mpv配置都没用，除非更改mpv或者ytdl的代码。
mpv应该是使用ffmpeg来读取网络流（？），目前windows版的mpv是内置了ffmpeg，不依赖外部的ffmpeg。

**Comment by dyphire** [comment]
楼上说的很清楚了
